code/datums/uplink/announcements.dm

```diff
@@ -87,15 +89,29 @@
 	item_cost = 2
 
 /datum/uplink_item/abstract/announcements/fake_ion_storm/get_goods(var/obj/item/device/uplink/U, var/loc)
-	ion_storm_announcement()
-	return 1
+	if(cooldown != 1)
+		ion_storm_announcement()
+		cooldown = 1
+		spawn(240)
+			cooldown = 0
+		return 1
+	else
+		loc << "<span class='danger'>This service is on cooldown! Try again in a bit!</span>"
+		return 0
 
 /datum/uplink_item/abstract/announcements/fake_radiation
 	name = "Radiation Storm Announcement"
 	desc = "Interferes with the station's radiation sensors. Triggers immediately upon investment."
 	item_cost = 6
```

Comment:
People get 25 TC now. Also don't merge. Extinguisher code is borked, working on a fix right now.